### PR TITLE
removed the token introspection command

### PR DESCRIPTION
**Pull request by bayer**
`hydra clients create` doesnt display the token anymore, so i removed the section dealing with the token

### CODE REVIEW

File: docs/hydra/5min-tutorial.md [hunk 1]
```diff
@@ -98,30 +102,6 @@ $ docker-compose -f quickstart.yml exec hydra \
     --client-id my-client \
     --client-secret secret
 
-UDYMha9TwsMBejEvKfnDOXkhgkLsnmUNYVQDklT5bD8.ZNpuNRC85erbIYDjPqhMwTinlvQmNTk_UvttcLQxFJY
```
Comment (aeneasr) [inline docs/hydra/5min-tutorial.md]: Please revert this change. The information shown here is correct and important.

Comment (bayer) [reply docs/hydra/5min-tutorial.md]: I dont't think the information is correct, since hydra does not display the token after the call - that's why I removed the token inspection command.

Comment (aeneasr) [reply docs/hydra/5min-tutorial.md]: The information is 100% correct, if it didn't show the token it was an error (such as misspelling, waiting too long before using the token, ...).

**Comment by aeneasr** [comment]
Thank you!
